core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowKeys.scala

```diff
@@ -72,5 +74,6 @@ trait CloudflowTaskKeys {
   private[sbt] val cloudflowBuildNumber = taskKey[BuildNumber]("The current Cloudflow build number (i.e. ${numberOfGitCommits}-${gitHeadCommit}).")
 
   private[sbt] val agentPaths = taskKey[Map[String, String]]("The paths to Java Agents added to Cloudflow application.")
+  private[sbt] val streamletsInProject = taskKey[Iterable[StreamletDescriptor]]("The streamlet descriptors")
```

Comment:
I think you can just use the existing key for streamlet descriptors.

Reply:
I don't see any existing key for `Iterable[StreamletDescriptors]` - hence built one ..

Reply:
why do you need a key

Reply:
private[sbt] val cloudflowStreamletDescriptors = taskKey[Map[String, Config]]("Streamlets found by scanning the application classpath.")